druid-shell/src/mac/mod.rs

```diff
@@ -88,10 +88,15 @@ impl<F: FnOnce(&dyn Any) + Send> IdleCallback for F {
 }
 /// This is the state associated with our custom NSView.
 struct ViewState {
+    nsview: WeakPtr,
     handler: Box<dyn WinHandler>,
     idle_queue: Arc<Mutex<Vec<Box<dyn IdleCallback>>>>,
 }
 
+struct WinCtxImpl<'a> {
+    nsview: &'a WeakPtr,
+}
```

Comment:
Future thought: we stash our host view, but I wonder if we won't also want to stash a reference to the window itself? we'll want to have access to this at some point.

Reply:
Why not just call NSView's [window](https://developer.apple.com/documentation/appkit/nsview/1483301-window?language=objc) method? I think that's what most native code would do.